client/login/wp-login/test/test.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { identity } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { Login } from '../index.jsx';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+describe( 'Login', () => {
+	useFakeDom();
+
+	describe( 'footerLinks', () => {
+		context( 'when state is not loaded', () => {
+			it( 'should not have a return button', () => {
+				const login = shallow( <Login queryArguments={ { redirect_to: '' } } translate={ identity } /> );
+
+				expect( login.find( '.wp-login__footer > a' ).length ).to.equal( 1 );
+			} );
+		} );
+
+		context( 'after state is loaded', () => {
+			context( 'there is no history', () => {
+				it( 'should not have a return button', () => {
+					const login = shallow( <Login queryArguments={ { redirect_to: '' } } translate={ identity } /> );
+					login.setState( { loaded: true } );
```

Comment:
That happens because in `render()` we have `PageViewTracker` which is a `connect`ed component, therefor it needs a redux store, so either we mock `PageViewTracker` or the redux store.